memcached.c

```diff
@@ -215,9 +215,10 @@ static rel_time_t realtime(const time_t exptime)
            underflow and wrap around to some large value way in the
            future, effectively making items expiring in the past
            really expiring never */
-        if (exptime <= process_started)
+        time_t expiration = exptime - process_started;
+        if (expiration <= 0 || expiration > (rel_time_t)(-1))
```

Comment:
```suggestion
        if (expiration <= 0 || expiration >= (rel_time_t)(-1))
```

expiration이 -1인 경우도 invalid value로 보아야 합니다.